scss/sections/_hero.scss

```diff
@@ -2,14 +2,10 @@
     background-image: url('../img/background/bgc-hero-XL.jpg');
     background-size: 100%;
     
-    .content {
+    .row {
```

Comment:
Co ty tu robisz? .row klasa jest zadeklarowana tylko raz w gridzie? Czemu ja nadpisujesz to? Czy zrozumiales czym jest wierz i kolumny?

Reply:
wciaz nic nie zmienione